python/openassetio/TraitBase.py

```diff
@@ -18,7 +18,7 @@
 """
```

Comment:
[consider] Update file docstring

Reply:
Good catch, thanks - updated [here](https://github.com/OpenAssetIO/OpenAssetIO/pull/706/commits/4cd7e7b3522768afae2cce58169e6ef4b4d11098)

Reply:
Actually, the second one didn't make sense, reverted [here](https://github.com/OpenAssetIO/OpenAssetIO/pull/706/commits/629ae4e965008d7efb39aa48ef9e55bd00e9da31).